lib/Interpreter/CppInterOp.cpp

```diff
@@ -422,10 +426,6 @@ namespace Cpp {
       return ND->getNameAsString();
     }
 
-    if (llvm::isa_and_nonnull<TranslationUnitDecl>(D)) {
-      return "";
-    }
-
```

Comment:
Why we had to move this?

Reply:
Actually thats an error from my hand reverting it back to original state

Reply:
Ok, then please undo that change.